photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -287,9 +291,9 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
 
     val evaluator =
       taskType match {
-        case LOGISTIC_REGRESSION =>
+        case LOGISTIC_REGRESSION | SMOOTHED_HINGE_LOSS_LINEAR_SVM =>
           new BinaryClassificationEvaluator(validatingLabelAndOffsets)
-        case LINEAR_REGRESSION =>
+        case LINEAR_REGRESSION | POISSON_REGRESSION =>
```

Comment:
After a discussion last time, we had decided to use the log likelihood as the best evaluator for Poisson Regression, given that it should always increase on the training data and generally increase on the test data as well. RMSE is clearly a bad metric, since Poisson regression does not try to fit the actual value of the predictor but a complicated function involving exp of the predictor.